CONTRIBUTING.md

```diff
@@ -0,0 +1,50 @@
+# How to contribute
+
+So you're thinking about contributing to NUnit? Great! Maintaining and enhancing NUnit is a big job, so **the community's help is really appreciated.**
```

Comment:
NUNit må byttes ut med Fhi.Frontend.....

Reply:
:man_facepalming: ...pinlig berørt, dette gikk litt for fort, litt for sent på dagen. Jeg håper og tror jeg vanligvis ikke er så sloppy :wink: